elastic/beats
# Source URI Reloading backports

Backport of multiple PRs related to config/source uri reloading

https://github.com/elastic/elastic-agent/pull/686
https://github.com/elastic/elastic-agent/pull/1252/

## How to test:
Standalone: 
- install
- run `./elastic-agent upgrade 8.9.10 --source-uri https://fake-uri.com

needs to fail with proper message

Managed
- install
- update policy to have custom source uri
- init upgrade
- see it fail with host not resolved

Succ scenario:
change uri to `http://localhost:8080` and start fileserver there
`python3 -m http.server`
directory needs to contain target asc, sha512 and archive files in location `beats/elastic-agent`
